debian/control

```diff
@@ -2,59 +2,55 @@ Source: gtk+3.0
 Section: libs
 Priority: optional
 Maintainer: Debian GNOME Maintainers <pkg-gnome-maintainers@lists.alioth.debian.org>
-Uploaders: Andreas Henriksson <andreas@fatal.se>, Michael Biebl <biebl@debian.org>
+Uploaders: Andreas Henriksson <andreas@fatal.se>, Michael Biebl <biebl@debian.org>, Sjoerd Simons <sjoerd@debian.org>
 Build-Depends: debhelper (>= 9.20141010),
                cdbs (>= 0.4.93),
-               quilt,
```

Comment:
Ah, I see where the quilt requirement was coming from now... yes, this needs to be changed. Never mind.